timestamp.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(express.static("./static"));
+
+app.get("/timestamp/:timeStamp", (req, res) => {
+    const TimeStamp = req.params.timeStamp;
+
+    console.log("timeStamp : "+TimeStamp);
+    let timeStamp = "-";
+
+    if(TimeStamp === "") {
+        timeStamp = (new Date()).getTime();
+        utcString = (new Date()).toUTCString();
+    }
+    else if ( !isNaN(TimeStamp) ) {
```

Comment:
is this what you are looking for?
```javascript
Number.isInteger(1555036282317)
```

Reply:
this wont work for integer. as number is beyond integer limits

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger#Examples